docs/advanced-usage/access-control/voms-proxy/index.md

```diff
@@ -0,0 +1,65 @@
+# Voms-proxy
+
+If your workflow needs special permissions that your Virtual Organization entitles you to, the VOMS client can create a proxy certificate for your authentication.
+
+## Testing your credentials
+
+Before moving to REANA, check that your credentials are in order.
+
+```console
+# Login to lxplus
+$ ssh johndoe@lxplus.cern.ch
+# Make sure userkey.pem is read/write only by the owner
+$ ls -l ~/.globus
+-rw-r--r--. usercert.pem
+-r--------. userkey.pem
+
+# Let us create a proxy certificate with the VO cms
+$ voms-proxy-init --voms cms
+Enter GRID pass phrase for this identity:
+
+Contacting voms2.cern.ch:15002 [/DC=ch/DC=cern/OU=computers/CN=voms2.cern.ch] "cms"...
+Remote VOMS server contacted succesfully.
+
+
+Created proxy in /tmp/x509up_u131816.
+
+Your proxy is valid until Wed Apr 01 00:43:08 CEST 2020
+```
+
+## Uploading secrets
+
+To create the proxy certificate the VOMS client needs access to your user certificate `usercert.pem`, encrypted private key `userkey.pem` and GRID pass phrase. For REANA the pass phrase needs to be encoded in Base64. To encode it exchange `mygridpassphrase` for your GRID pass phrase and perform
```

Comment:
I don't know if we should rephrase the part that says "the VOMS client needs access to your user certificate" to "REANA needs access to your user certificate", what do you think? For me sounds a bit too much to give details about how it is implemented in the docs. However, I kind of understand it in this case, the users would give their credentials and it is nice to know what part of the system is using it (they are familiar with VOMS client nomenclature).

Reply:
You are right, it does sound better to only mention REANA. I think my thought process was that people are used to the VOMS client and I need to specify usually the client needs this but for REANA we need the pass phrase in Base64. But I agree with you :)